specs/binding-grpc.spec/src/main/scripts/io/aklivity/zilla/specs/binding/grpc/schema/grpc.schema.patch.json

```diff
@@ -124,6 +131,40 @@
                                     }
                                 },
                                 "additionalProperties": false
+                            },
+                            "catalog":
+                            {
+                                "title": "Catalog",
+                                "type": "object",
+                                "patternProperties":
+                                {
+                                    "^[a-zA-Z]+[a-zA-Z0-9\\._\\-]*$":
+                                    {
+                                        "type": "array",
+                                        "items":
+                                        {
+                                            "type": "object",
+                                            "properties":
+                                            {
+                                                "subject":
+                                                {
+                                                    "type": "string"
+                                                },
+                                                "version":
+                                                {
+                                                    "type": "string",
+                                                    "default": "latest"
+                                                }
+                                            },
+                                            "required":
+                                            [
+                                                "subject"
+                                            ],
+                                            "additionalProperties": false
```

Comment:
```suggestion
                                            "$ref": "#/$defs/cataloged"
```
?

Reply:
This change was intentional since the only binding that uses the global catalog definition is the `grpc` server and it doesn't support all of the options defined in the `#/$defs/cataloged` object. 

Unless it does support those, in which case, I will add them to the docs.

Reply:
I think it is supposed to defer cataloged parsing to a generic parser instead of redoing it locally.
I'll double check with @akrambek .